CODEOWNERS

```diff
@@ -24,4 +24,4 @@
 
 # The following lines are used by GitHub to automatically recommend reviewers.
 
-* @abertelrud @neonichu @tomerd @friedbunny
+* @abertelrud @neonichu @tomerd @elsh
```

Comment:
Intentional replacement?

Reply:
added. I think it makes sense to keep Daniel and Ankit info in that section since they show up in commit history, correct?

Reply:
> Intentional replacement (summary says to add)?

yes I believe @friedbunny would be less involved in these going forward